dspace-api/src/main/java/org/dspace/app/itemimport/ItemImportServiceImpl.java

```diff
@@ -725,7 +725,14 @@ protected void addDCValue(Context c, Item i, String schema, Node n) throws Trans
         String qualifier = getAttributeValue(n, "qualifier"); //NodeValue();
         // //getElementData(n,
         // "qualifier");
-        String language = getAttributeValue(n, "language");
+
+        String language = null;
+
+        //DS-4493 protection against initialising as an empty string
+        if (!"".equals(getAttributeValue(n, "language"))){
```

Comment:
I would suggest something like lang3.isBlank rather than just comparing with "", to catch whitespace of nonzero length.  I think I've actually seen this happen.

(Separately, ought we to have Metadatavalue require a non-null language to be a valid ISO 639-1 code?)

Reply:
+1 on using isNotBlank(). Something similar was done in https://github.com/DSpace/DSpace/pull/2350/files.

Reply:
@bram-atmire : I agree with the feedback of @mwoodiupui and @nwoodward here.  This should be updated.  It also looks like it may need porting to `master`.

Reply:
Thanks @mwoodiupui @nwoodward @tdonohue , agreed & resolved.